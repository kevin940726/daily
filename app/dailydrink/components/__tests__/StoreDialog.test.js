const { html } = require('../../../slack-block-kit');
const StoreDialog = require('../StoreDialog');

test('it renders <StoreDialog />', () => {
  const block = html`
    <${StoreDialog} />
  `;

  expect(block).toMatchSnapshot();
});
