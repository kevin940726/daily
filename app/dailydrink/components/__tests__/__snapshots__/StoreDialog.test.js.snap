// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders <StoreDialog /> 1`] = `
Object {
  "callback_id": "create-new-store-callback-id",
  "elements": Array [
    Object {
      "label": "Store name",
      "name": "storeName",
      "type": "text",
    },
    Object {
      "label": "Menu image URL",
      "name": "imageURL",
      "subtype": "url",
      "type": "text",
    },
  ],
  "notify_on_cancel": false,
  "submit_label": "Submit",
  "title": "Submit a new store",
}
`;
